torch/_dynamo/symbolic_convert.py

```diff
@@ -3936,7 +3936,7 @@ def inline_call_(self):
             ):
                 assert isinstance(self, InliningGeneratorInstructionTranslator)
                 # When the generator returns None, we raise StopIteration
-                exc.raise_observed_exception(StopIteration, self)
+                exc.raise_observed_exception(StopIteration, self, args=[self.symbolic_result], observed_exception_args=[self.symbolic_result])
```

Comment:
```suggestion
                exc.raise_observed_exception(StopIteration, self, args=[self.symbolic_result])
```

Reply:
Looks reasonable! (I was wondering because that changes other calls to here; but if CI passes then that looks good)